app/src/views/kyc/tier2/steps/sumSubInjectStyle.txt

```diff
@@ -0,0 +1,203 @@
+:root {
```

Comment:
Why is this a .txt? If you need css but css-modules cause troubles, wrap everthing in `:global {...`

Reply:
This are styles that I have to send into an <iframe> passing them as a JSON parameter in SumSubSDK configuration. So Webpack can't touch or parse this styles.